modules/assim.batch/R/pda.emulator.R

```diff
@@ -472,16 +517,20 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
   
   
   if (FALSE) {
-    gp     <- kernlab.gp
-    x0     <- init.x
-    nmcmc  <- settings$assim.batch$iter
-    rng    <- NULL
-    format <- "lin"
-    mix    <- "each"
-    jmp0   <- apply(X, 2, function(x) 0.3 * diff(range(x)))
-    jmp0   <- sqrt(unlist(settings$assim.batch$jump$jvar))
-    ar.target <- settings$assim.batch$jump$ar.target
-    priors <- prior.fn$dprior[prior.ind]
+     gp          = gp
+     x0          = init.list[[chain]]
+     nmcmc       = settings$assim.batch$iter
+     rng         = rng
+     format      = "lin"
+     mix         = mix
+     jmp0        = jmp.list[[chain]]
+     ar.target   = settings$assim.batch$jump$ar.target
+     priors      = prior.fn.all$dprior[prior.ind.all]
+     settings    = settings
+     run.block   = (run.normal | run.round)  
+     n.of.obs    = n.of.obs
+     llik.fn     = llik.fn
+     resume.list = resume.list[[chain]]
```

Comment:
What is the purpose of this block? I'm assuming it's for interactive testing (which is smart! I'll have to start using that), and if so, it might be good to add a comment indicating that.

Also, I've taken a liking to using `interactive()` (no arguments), which returns `TRUE` if you're running interactively and `FALSE` if you're running via script. It's probably not appropriate here, but just thought I'd call your attention to it.